### PR TITLE
Added another Event Set for Receiving I2C MIDI Events

### PR DESCRIPTION
**Pull request by sennert**
added another I2C Event Set for receiving MIDI Events (max 3 Bytes) via I2C. Tested and Working

### CODE REVIEW

File: automat/automat.ino [hunk 1]
```diff
@@ -441,6 +442,25 @@ void receiveI2CEvent(int len)
...
+        uint8_t byte2 = Wire.read();
+        uint8_t byte3 = Wire.read();
+
+        uint8_t header = byte1 & 0xF0;
+        switch (header) {
+          case 0x80:  // Note-off
+              handleNoteOff(1 + (byte1 & 0x0F), byte2, byte3);
+              break;
+          case 0x90:  // Note-on
+              handleNoteOn(1 + (byte1 & 0x0F), byte2, byte3);
```
Comment (lefty71) [inline automat/automat.ino:457]: MIDI Note-on messages with a velocity of 0 should be treated as a note-off event.

Comment (lefty71) [reply automat/automat.ino:457]: So if byte3 == 0 for the 0x90 case, you should call handleNoteOff(...)

Comment (sennert) [reply automat/automat.ino:457]: thanks lefty71. of course. I'm already filtering those zero velocity events at the sender side :)

Comment (sennert) [reply automat/automat.ino:457]: added a new pull request with filter also included on receiver side :)

**Comment by sennert** [comment]
added

**Pull request closed without merging**